### PR TITLE
Prepping for 7-5-2017 Release Notes update

### PR DESCRIPTION
**Pull request by davidgardner11**
Prepping for 7-5-2017 Release Notes update

### CODE REVIEW

File: Release Notes/2017/2017-07-06-cloud-platform-release-notes.md [hunk 1]
```diff
@@ -0,0 +1,67 @@
...
+
+  - __Cloudportal.io Access__
+
+    Enabled access for cloudportal.io accounts.
+
+### Ecosystem Partner Integrations (2)
+
+* __CenturyLink Marketplace Provider Integrations__
+
+  - __[Alert Logic Threat Manager with ActiveWatch](https://www.ctl.io/marketplace/partner/PST/product/Alert%20Logic%20Threat%20Manager%20with%20ActiveWatch/)__ is a cloud-based managed network intrusion detection and vulnerability assessment solution delivered as-a-service that works in any datacenter environment, from on-premises to the cloud. It works the same way in every environment, so you can keep costs down without having to learn multiple systems and hire additional staff.
```
Comment (srveit) [inline Release Notes/2017/2017-07-06-cloud-platform-release-notes.md]: Add `__` after `ActiveWatch]`

Comment (davidgardner11) [inline Release Notes/2017/2017-07-06-cloud-platform-release-notes.md]: good catch. I'll add that

Comment (davidgardner11) [inline Release Notes/2017/2017-07-06-cloud-platform-release-notes.md]: ok. i figured it out. It does have the `__` -- its just after the hyperlink.

Comment (srveit) [inline Release Notes/2017/2017-07-06-cloud-platform-release-notes.md]: I see it now.

**Pull request merged**